pkg/controller/podautoscaler/horizontal.go

```diff
@@ -494,23 +504,10 @@ func (a *HorizontalController) reconcileAutoscaler(hpav1Shared *autoscalingv1.Ho
 			rescaleReason = "All metrics below target"
 		}
 
-		desiredReplicas = a.normalizeDesiredReplicas(hpa, currentReplicas, desiredReplicas)
-
-		rescale = a.shouldScale(hpa, currentReplicas, desiredReplicas, timestamp)
-		backoffDown := false
-		backoffUp := false
-		if hpa.Status.LastScaleTime != nil {
-			if !hpa.Status.LastScaleTime.Add(a.downscaleForbiddenWindow).Before(timestamp) {
-				setCondition(hpa, autoscalingv2.AbleToScale, v1.ConditionFalse, "BackoffDownscale", "the time since the previous scale is still within the downscale forbidden window")
-				backoffDown = true
-			}
-		}
-
-		if !backoffDown && !backoffUp {
-			// mark that we're not backing off
-			setCondition(hpa, autoscalingv2.AbleToScale, v1.ConditionTrue, "ReadyForNewScale", "the last scale time was sufficiently old as to warrant a new scale")
-		}
-
+		desiredReplicas = a.normalizeDesiredReplicas(hpa, key, currentReplicas, desiredReplicas)
+		// mark that we're not backing off
+		setCondition(hpa, autoscalingv2.AbleToScale, v1.ConditionTrue, "ReadyForNewScale", "the last scale time was sufficiently old as to warrant a new scale")
```

Comment:
this message is no longer valid, since we don't have backoff any more.  You'll want to adjust the condition reasons, and add new condition reasons for when the max in the window is larger than the most recent data point

Reply:
Done.